src/config/config.c

```diff
@@ -99,19 +99,19 @@ typedef enum {
 } CONFIG_ACTION;
 
 /*
+ * Some inspiration taken from https://github.com/quartzjer/js0n (public domain). These
```

Comment:
Where did this comment come from? It doesn't seem like part of the PR

Reply:
Yeah, I was talking to Michael about this code and it turns out there used to be a comment giving credit, but it went missing at some point. So I figured while I was here I'd reinstate it.